modules/channel-web/src/backend/db.ts

```diff
@@ -156,7 +157,7 @@ export default class WebchatDb {
     userId: string,
     conversationId: number,
     payload: any,
-    incomingEventId: string,
+    eventId: string,
```

Comment:
We should keep both, this way it's easier to associate messages. Currently in the debugger when you select a message, we use the incomingEventId to know which event to load when you select an outgoing message

Reply:
I totally agree that shouldn't not be removed totally a mistake. Agreed

Reply:
Ah just realized, both are the same here it's appendUserMessage method which is an incoming event , in this case incomingEventId and eventId is the exact same thing so nothin to be done here